docs/GoogleDrive-batchsink.md

```diff
@@ -0,0 +1,26 @@
+# Google Drive Batch Sink
+
+
+Description
+-----------
+Sink plugin saves files from the pipeline to Google Drive directory via Google Drive API.
+The minimal input schema contains only byte array with field name configured by **schemaBodyFieldName** property. 
+In this case files have rangomly generated 16-symbols names.
+Also file name can be retrieved from input scheme by configuring of **schemaNameFieldName** property.
+
+Properties
+----------
+
+**clientId** OAuth2 client id.
```

Comment:
In doc you use label names to variable names.

Reply:
done